perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -47,7 +47,8 @@ git fetch origin -q
 
 function execute_perf_test() {
   mkdir -p gcs
-  GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100"
+  echo "running perf tests with trace logs"
```

Comment:
Assuming all these are temporary changes for presubmit perf test. Adding a note to revert before merge.

Reply:
Reverted this. The results are in the PR description.